pkg/legacy/converter.go

```diff
@@ -217,3 +218,33 @@ func buildConfigProviders(agentConfig Config) ([]config.ConfigurationProviders,
 
 	return []config.ConfigurationProviders{cp}, nil
 }
+
+func buildHistogramAggregates(agentConfig Config) []string {
+	configValue := agentConfig["histogram_aggregates"]
+
+	var histogramBuild []string
+	var result []string
+	validValues := [6]string{"min", "max", "median", "avg", "sum", "count"}
```

Comment:
you don't need the `6` and could you add a quick comment saying that those value are the available ones in agent5 ?